Entities/ACrab.h

```diff
@@ -585,6 +597,7 @@ int FirearmActivationDelay() const;
 	float m_JetTimeTotal;
 	// How much time left the jetpack can go, in ms
 	float m_JetTimeLeft;
+	float m_JetReplenishRate; //!< How fast the jetpack fuel will replenish when not in use, in relation to the legacy value.
```

Comment:
Referencing "the legacy value" isn't super meaningful imo. I'd say it's more important that it's clear that 1.0 is "normal" speed, 2 is double, 0 is no replenishment, etc.

Reply:
But literally the only "normal" thing about the legacy value is that it's the legacy value lol

The "normal" value *is* double speed in relation to how fast the fuel depletes, which is why I'd rather just have `2.0F` be the default value (I only changed it to 1.75 to re-balance the fast regen)

Reply:
Yeah, I get ya. This one's a pain - it's normal not for any good reason except cause it is lol
Maybe it'd be better to do the arduous thing and remove the hardcoded `2.0`, default the value to `1.0` and add the property in ini everywhere. This'd screw with mods, but I guess the converter could probably be set up to add some value in to every mod it converts.

Going back to how you had it originally, I wonder if something like `JetReplenishAmount` or `JetReplenishPerUpdate` would better convey that it's defining the absolute amount the jetpack is replenished by? Conversely, I wonder if it should be `JetReplenishRateMultiplier` to convey that it's a multiplier for jetpack replenishment?
Gawd, naming this one is hard :( You check with maxim on it (maybe I asked you this already)?

Reply:
@MaximDude